### DISCUSSION TITLE
Is there any way to drop layout inside layouts

### DISCUSSION THREAD

**Discussion by Rahul139-stack** [websight-io/starter#112] [Q&A] [answered]
When we trying to drop 1 cols layout inside 2 cols layouts its not working.

**Comment by paweljozwicki** [comment, marked as answer]
Hi, could you provide some more details? Do you work on one of our components libraries or create your own components based on primeflex or something like that?

**Comment by Rahul139-stack** [comment]
Though we tried the howlite layouts, we tried to drag 1 columns layout inside 2 columns layout, so it is not working. Is there any suggestion to achieve this workable.
 **Sample** 
Here we drag the the 2 cols layout and just tried to drag and drop one cols layout inside 2 cols layout.
![MicrosoftTeams-image (12)](https://user-images.githubusercontent.com/81350384/210737126-2d79f70e-4e73-4167-b8eb-05754a271640.png)


 Is there any layouts to add side bar to the page?

**Reply by paweljozwicki** [reply]
I see. According to the [documentation](https://www.websight.io/docs/authors/component-libs/howlite/layouts/), those layouts are page sections that have to speed up building whole pages. If you want to play with adding more columns inside then you have to add some containers manually and change Grid properties. Those layouts can be added only to the page. I think that their names are misleading and we should consider changing them.

**Reply by Rahul139-stack** [reply]
@paweljozwicki Thanks
